tests/test_views.py

```diff
@@ -2848,25 +2848,6 @@ def _make_pointer_only_user_can_see(self, user, project, save=False):
         node = ProjectFactory(creator=user)
         project.add_pointer(node, auth=Auth(user=user), save=save)
 
-    def test_pointer_list_write_contributor_can_remove_private_component_entry(self):
```

Comment:
For posterity. Per a meeting with @mfraezz @brianjgeiger and I, it was determined that in the component widget, we would not admit in any way that a private link exists if the user does not have read access to the node referenced by that link.

  As long as that is the case this test can't pass. Unless there is an alternative target for the test.